legion/engine/rendering/components/light.hpp

```diff
@@ -12,7 +12,7 @@ namespace legion::rendering
 
     namespace detail
     {
-        struct light_data
+        struct [[reflectable]] light_data
```

Comment:
does this generate correctly? i think in this case the preprocessor doesn't know the namespace of the attribute right?